### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by Aman-Jain1996**
Added cart page

### CODE REVIEW

File: src/pages/Product-Details/Product-details.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  border-radius: 8px;
+}
+
+.product-details .content {
+  flex: 0 0 50%;
+  padding: 0 4.8rem;
+}
+
+.product-name {
+  color: #000;
```
Comment (Hnh013) [inline src/pages/Product-Details/Product-details.css:33]: A suggestion : if feasible , for future iterations use variables for color , instead hard coding.

**Review by Hnh013 — APPROVED** [review]
Added one small suggestion , rest all looks good , code is structured and clean .

File: src/components/NavBar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+.Nav-header {
+  display: flex;
+  align-items: center;
+  background-color: var(--bg-lightWhite);
+  height: 12vh;
+  padding: 2.4rem 4.8rem;
+  justify-content: space-between;
+  position: sticky;
+  top: 0;
+  /* box-shadow: 0 4px 12px 0 rgba(0 , 0 ,0 , .8); */
```
Comment (Rohanmond) [inline src/components/NavBar/Navbar.css:12]: remove unwanted comments

File: src/pages/Cart/Cart.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import Navbar from "../../components/NavBar/Navbar";
+import "./Cart.css";
+import productImage from "../../assets/images/product1.webp";
+
+const Cart = () => {
+  return (
+    <div>
+      <Navbar />
+      <h3 class="main-heading">My Cart (2)</h3>
```
Comment (Rohanmond) [inline src/pages/Cart/Cart.jsx:10]: (2) number can be dynamic

**Review by Rohanmond — APPROVED** [review]

**Pull request merged**